### DISCUSSION TITLE
Can ArchR integrate scATAC-seq from different platforms?

### DISCUSSION THREAD

**Discussion by yding9** [GreenleafLab/ArchR#591] [Questions / Documentation] [answered]
Hello,

I'm using ArchR for 10X scATAC-seq and multiome analysis and find out the significant difference between the two datasets (scATAC-seq and multiome). In our institute, we do need to integrate datasets from different technologies after dataset-specific differences removed. I notice that Seurat's integration methods could deal with this effect (https://satijalab.org/signac/articles/integration.html). I wonder if ArchR has some similar functions or a wrapper of Seurat's integration methods available for use? Or could you provide me some instructions/codes to output ArchR's intermediate results and import them into Seurat's FindIntegrationAnchors and/or IntegrateData functions? 

Your help is highly appreciated!

**Comment by rcorces** [comment, marked as answer]
Thanks for using Discussions!

We dont have anything like this implemented (beyond using Harmony which isnt really the same). Curious to know if @jgranja24 has any thoughts on this.

**Comment by yding9** [comment]
@rcorces @jgranja24 Thank you for letting me know! Curious to know if you are planning support for integration.

**Reply by rcorces** [reply]
This isnt on our list of things that are high priority but it seems reasonable to implement. Have you tried using the tools already available through ArchR though? Are you sure you cannot accomplish something similar with Harmony? For example, in our paper we integrate PBMC data from many different platforms without too many difficulties.

**Reply by yding9** [reply]
Glad to hear there is a chance this might be implemented. Yes, I have tried Harmony through ArchR. But Harmony doesn't fully resolve the batch effects from these two platforms though it makes them closer. I was impressed by your PBMC data but I wasn't that lucky to get something similar with Harmony alone. At our institute, we use CCA to map scRNA-seq from different platforms so we wonder if we could apply CCA or something similar on scATAC data.
